vault/logical_system_namespaces.go

```diff
@@ -193,6 +230,12 @@ func (b *SystemBackend) handleNamespacesRead() framework.OperationFunc {
 func (b *SystemBackend) handleNamespacesSet() framework.OperationFunc {
 	return func(ctx context.Context, req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
 		path := data.Get("path").(string)
```

Comment:
It is slightly confusing that we don't do the validation here, but handle it in the namespace store; should we add a comment?

Reply:
Sorry, guess this comment was illegible :D 

I meant the validation that all parents exist, and we don't create foo/bar/baz without foo/bar existing, if our request sets bar/baz as path.

Reply:
Based on the feedback from you and @driif, I added checks to all the handlers that accept a `:path` URL param that forbids `/` in the paths. This means, that we can now rely on the parent namespace existing for sure in context inside the handlers (`(*NamespaceStore).ResolveNamespaceFromRequest` would otherwise fail before), so I don't think any more checking is necessary here.